hack/update-rhcos-bootimage.py

```diff
@@ -23,6 +23,12 @@
     string_f = codecs.getreader('utf-8')(f)  # support for Python < 3.6
     meta = json.load(string_f)
 newmeta = {}
+
+# x86_64 boot image bumps should contain more than just the ostree tarball and
+# qemu qcow2 image
+if args.arch == 'amd64' and len(meta['images']) <= 2:
+    raise SystemExit("You appear to be missing most of the x86_64 boot image "
+                     + "artifacts. Check the meta.json for this RHCOS version.")
```

Comment:
While we're here, we might as well say something like "Check that full artifact generation was enabled for this build", or whatever is the appropriate terminology.